samples/NHSD.BuyingCatalogue.Identity.Api.SampleMvcClient/Startup.cs

```diff
@@ -22,6 +22,9 @@ public void ConfigureServices(IServiceCollection services)
         {
             services.AddControllersWithViews();
 
+            var clientId = Configuration.GetSection("clientId").Value;
```

Comment:
Can you review the use of these variables? I can't see them used elsewhere in the method.